test-suite/shared/src/test/scala/org/scalajs/testsuite/junit/JUnitAssertionsTest.scala

```diff
@@ -326,4 +326,81 @@ class JUnitAssertionsTest {
     testIfAsserts(assertThat(0, instanceOf[Int](classOf[Double])), ShallNotPass)
   }
 
+  @Test
+  def testExpectThrows(): Unit = {
+    testIfAsserts(expectThrows(classOf[Exception],
+        throwingRunnable { throw new Exception }))
+    testIfAsserts(expectThrows(classOf[IndexOutOfBoundsException],
+        throwingRunnable { throw new IndexOutOfBoundsException }))
+
+    testIfAsserts {
+      val ex = expectThrows(classOf[Exception],
+          throwingRunnable { throw new Exception("abc") })
+      assertEquals(ex.getMessage, "abc")
+    }
+
+    testIfAsserts(expectThrows(classOf[IndexOutOfBoundsException],
+        throwingRunnable { throw new Exception }), ShallNotPass)
+    testIfAsserts(expectThrows(classOf[IndexOutOfBoundsException],
+        throwingRunnable { }), ShallNotPass)
+  }
+
+  @Test
+  def testAssertThrows(): Unit = {
+    testIfAsserts(assertThrows(classOf[Exception],
+        throwingRunnable { throw new Exception }))
+    testIfAsserts(assertThrows(classOf[IndexOutOfBoundsException],
+        throwingRunnable { throw new IndexOutOfBoundsException }))
+
+    testIfAsserts(assertThrows(classOf[IndexOutOfBoundsException],
+        throwingRunnable { throw new Exception }), ShallNotPass)
+    testIfAsserts(assertThrows(classOf[IndexOutOfBoundsException],
+        throwingRunnable { }), ShallNotPass)
+  }
+
+  /** Dummy implementation of Assert.assertThrows to be used until JUnit 4.13 is
+   *  released. See org.junit.Assert.scala in jUnitRuntime.
+   */
+  private def assertThrows(expectedThrowable: Class[_ <: Throwable],
```

Comment:
We should put these in `org.scalajs.testsuite.util.AssertThrows` or something.
Also, they're *not* "dummy", that's not the right word. They're real implementations since they actually do their job.